### PR TITLE
allow creation of bigquery datasets for cluster usage monitoring

### LINKED ISSUE #91: Add cluster usage metering
Google offers the possibility to monitor cluster usage by creating a BigQuery dataset and attach it to the cluster.

The documentation from Google side is mentioned here: https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-usage-metering

### PR DESCRIPTION
**Pull request by boxcee**
Closes #91

### CODE REVIEW

File: modules/gke-cluster/variables.tf [hunk 1]
```diff
@@ -149,6 +149,24 @@ variable "alternative_default_service_account" {
...
+variable "bigquery_dataset_name" {
+  description = "Name of the BigQuery dataset to collect cluster usage data."
+  type        = string
+  default     = ""
+}
+
+variable "bigquery_dataset_location" {
+  description = "Location of the BigQuery dataset."
+  type        = string
+  default     = "EU"
```
Comment (robmorgan) [inline modules/gke-cluster/variables.tf:167]:
```suggestion
  default     = "US"
```
I would suggest changing this to the US as that's where the bulk of our users are.

**Comment by robmorgan** [comment]
Hi @boxcee,

Thanks for your PR. I've left some comments in my review. The only thing that is missing is some tests. Could you add tests using [Terratest](https://terratest.gruntwork.io) that pass values into the 3 new variables you have created? That way we can properly assert the behavior is working as as expected.

**Comment by robmorgan** [comment]
Hi @boxcee did you still want to contribute the changes to this PR?

**Comment by boxcee** [comment]
Yes. Will resolve today.

**Pull request closed without merging**